contracts/root/withdrawManager/WithdrawManager.sol

```diff
@@ -31,6 +32,73 @@ contract WithdrawManager is WithdrawManagerStorage /* , IWithdrawManager */ {
     _;
   }
 
+  /**
+   * @notice Start an exit from the side chain by referencing the preceding (reference) transaction
+   * @param referenceData RLP encoded data of the reference tx that encodes the following fields:
+   * headerNumber Header block number of which the reference tx was a part of
+   * blockProof Proof that the block header (in the child chain) is a leaf in the submitted merkle root
+   * blockNumber Block number of which the reference tx is a part of
+   * blockTime Reference tx block time
+   * blocktxRoot Transactions root of block
+   * blockReceiptsRoot Receipts root of block
+   * receipt Receipt of the reference transaction
+   * receiptProof Merkle proof of the reference receipt
+   * branchMask Merkle proof branchMask for the receipt
+   * @param logIndex Log Index to read from the receipt
+   * @param exitTx Signed exit transaction
+   */
+  function startExit(
+    bytes memory referenceData,
+    uint8 logIndex,
+    bytes memory exitTx)
+    public
+  {
+    RLPReader.RLPItem[] memory referenceTxData = referenceData.toRlpItem().toList();
+    (address childToken, address rootToken, uint256 closingBalanceOrTokenId, /* bool isErc721 gives stack too deep */) =
+    ChildChainVerifier.processReferenceTx(
+      bytes32(referenceTxData[5].toUint()), // blockReceiptsRoot,
+      referenceTxData[6].toBytes(), // receipt
+      referenceTxData[7].toBytes(), // receiptProof
+      referenceTxData[8].toBytes(), // branchMask
+      logIndex,
+      msg.sender,
+      address(registry)
+    );
+    require(
+      registry.rootToChildToken(rootToken) == childToken,
+      "INVALID_ROOT_TO_CHILD_TOKEN_MAPPING"
+    );
+
+    // validate exitTx
+    (uint256 exitAmountOrTokenId, bool burnt) = ExitTxValidator.processExitTx(exitTx, childToken, msg.sender);
+    // gives stack too deep, fix later
```

Comment:
Fix stack too deep :)

Reply:
Have fixed this along with some changes in the next PR.